nj-build/src/lib.rs

```diff
@@ -1,20 +1,22 @@
+mod win_delay_load_hook;
+
 /// Slightly modified from https://github.com/Brooooooklyn/napi-rs/blob/master/build/src/lib.rs
 /// configure linker to generate node.js dynamic library
 #[cfg(windows)]
 pub fn configure() {
-        
     // On Windows, we need to download the dynamic library from the nodejs.org website first
     use std::env::var;
     use std::fs::File;
     use std::io::copy;
     use std::path::PathBuf;
     use std::process::Command;
 
-    let node_full_version = String::from_utf8(Command::new("node").arg("-v").output().unwrap().stdout).unwrap();
-        
-    let temp_dir =   PathBuf::from("C:\\Windows\\Temp\\");
-    let mut temp_lib =   temp_dir.clone();
-    
+    let node_full_version =
+        String::from_utf8(Command::new("node").arg("-v").output().unwrap().stdout).unwrap();
+
+    let temp_dir = PathBuf::from("C:\\Windows\\Temp\\");
```

Comment:
can you incorporate these changes I made for another branch to get rid of hard coded temp directory?

```
let node_full_version = String::from_utf8(Command::new("node").arg("-v").output().unwrap().stdout).unwrap();
        
    let temp_lib =   temp_dir().join(format!("node-{}.lib", node_full_version.trim_end()));

    if !temp_lib.exists() {

        let lib_file_download_url = format!(
            "https://nodejs.org/dist/{}/win-x64/node.lib",
            node_full_version
        );

        println!("downloading nodejs: {} to: {:#?}",lib_file_download_url,temp_lib);

        let mut resp =
            reqwest::blocking::get(&lib_file_download_url).expect("Download node.lib file failed");
        let mut node_lib_file = File::create(&temp_lib).unwrap();
        copy(&mut resp, &mut node_lib_file).expect("Save node.lib file failed");
    }
```

Reply:
Yeah

Reply:
do you have a definition for `temp_dir()` somewhere?

Reply:
use std::env::temp_dir;